jumpscale/sals/nginx/static/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>index</title>
+<style>
+    
+body {
+    margin: 0 auto;
+    font-family: Tahoma, Verdana, Arial, sans-serif;
+    background-image: url("/images/background.jpg");
+    background-repeat: no-repeat;
+    background-position: center center;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+    color: black;
+    height: 100vh;
+
+
+}
+.button {
+    background-color: #4CAF50;
+    border: none;
+    color: white;
+    padding: 15px 32px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: 4px 2px;
+    cursor: pointer;
+}
+.center {
+   width: 200px;
+   height: 600px;
+   position: absolute; /*Can also be `fixed`*/
+   left: 0;
+   right: 0;
+   top: 0;
+   bottom: 0;
+   margin: auto;
+   /*Solves a problem in which the content is being cut when the div is smaller than its' wrapper:*/
+   max-width: 100%;
+   max-height: 100%;
+   overflow: auto;
+}
+.h-center {
+  margin-top:60px;
+  width: 50%;
+  margin: 0 auto;
+}
+.large-font{
+    font-size: 300%;
+    text-align: center;
+}
+}
+
+</style>
+</head>
+<body>
+    <div class="h-center">
+        <p class="large-font">welcome to 3Bot go to your admin dashboard from here</p>
```

Comment:
Can we adjust this typo a little starts with capital letter

```suggestion
        <p class="large-font">
            Welcome to 3Bot
        </p>
```

Reply:
done